src/index.html

```diff
@@ -15,8 +15,106 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&amp;display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
```

Comment:
The `header` tag is a semantic tag, which is good. However, the content inside the `header` is wrapped in a `div` tag which is not semantic. Consider using a more semantic tag, like `section` or `nav`, if it is appropriate.